src/components/AddNewCommentForm.jsx

```diff
@@ -72,21 +72,34 @@ class AddNewCommentForm extends React.Component {
 
     init() {
         // Get the settings (it won't make a network call as the data is cached!
-        this.customizrClient
-            .getSettings(this.props.accessToken)
-            .then((json) => {
-                let newAlertDismissed = json.mentionsUsageNotification && json.mentionsUsageNotification.alertDismissed === true;
-                let newSelectedVisibilityOption = this.state.commentVisibilityLevels.find((l) => l.value === json.selectedVisibility);
-                // Only update the state if there is change
-                if (this.state.alertDismissed !== newAlertDismissed || this.state.selectedVisibilityOption !== newSelectedVisibilityOption) {
-                    this.safeSetState({
-                        alertDismissed: newAlertDismissed,
-                        selectedVisibilityOption: newSelectedVisibilityOption,
-                    }, () => {
-                        this.resetSelectedVisibilityOption();
-                    });
-                }
-            });
+        if (this.props.enforceVisibilityLevel) {
+            let newSelectedVisibilityOption = this.state.commentVisibilityLevels.find((l) => l.value === this.props.enforceVisibilityLevel);
+            // Only update the state if there is change
+            if (this.state.selectedVisibilityOption !== newSelectedVisibilityOption) {
+                this.safeSetState({
+                    alertDismissed: true,
```

Comment:
This will basically make sure that users never see the @ mention info if the `this.props.enforceVisibilityLevel` is set